Client/src/Assets/readMeMessage.tsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react'
+
+export const rm = "To properly upload a JSON file based on the provided empty sample, \nthe user must fill up the following fields: \n" + "-reference \n-type \n-publisher \n-author \n-title \n-year \n-dataset/materials \n-variables/points \n" + "In order to enter the point array values, there should be a minimum of one value per variable: ie. \n[density, temp, pressure, init velocity, shock velocity, particle, velocity, compression, etc.] \n" + "NOTE:  volume, pages, datePublished, dateAccessed, details {of materials}, repr, units, subcategory, and data type are optional fields whose lines can/should be removed if not applicable.";
```

Comment:
doi is a required field

Change the last sentence to this for clarity:
"NOTE the following are optional fields and their lines should be removed if not applicable: subcategory, data type, [in reference:] volume, pages, datePublished, dateAccessed, [in materials:] details, and [in variables:] repr, units."

Reply:
updated and fixed. Thank you sean!